src/runtime/virtcontainers/pkg/cgroups/cgroups.go

```diff
@@ -6,24 +6,22 @@
 package cgroups
 
 import (
-	"os"
+	"fmt"
 	"path/filepath"
-	"sync"
 
-	"github.com/containerd/cgroups"
 	v1 "github.com/containerd/cgroups/stats/v1"
 	"github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/sirupsen/logrus"
 )
 
-type Cgroup struct {
-	cgroup  cgroups.Cgroup
-	path    string
-	cpusets *specs.LinuxCPU
-	devices []specs.LinuxDeviceCgroup
+// prepend a kata specific string to oci cgroup path to
+// form a different cgroup path, thus cAdvisor couldn't
+// find kata containers cgroup path on host to prevent it
+// from grabbing the stats data.
+const CgroupKataPrefix = "kata"
```

Comment:
Since it's only ever used by `cgroups.go` I thought it would be a good place for it.
I moved the `const DefaultResourceControllerID = "/vc"` definition to `utils_linux.go` though, as it's used outside of the package.